website/content/en/docs/building-operators/golang/project_migration_guide.md

```diff
@@ -100,14 +100,14 @@ operator-sdk create api \
 
 ### API's
 
-Now let’s copy the API definition from `pkg/apis/<group>/<version>/<kind>_types.go` to `api/<version>/<kind>_types.go`. For our example, it is only required to copy the code from the `Spec` and `Status` fields. 
+Now let’s copy the API definition from `pkg/apis/<group>/<version>/<kind>_types.go` to `api/<version>/<kind>_types.go`. For our example, it is only required to copy the code from the `Spec` and `Status` fields.
 
-Observer that this file is quite similar to the old one. You ought to copy all for your new API and it will be pretty much verbatim, however, it requires some attention with the [Markers][markers]:
+This file is quite similar to the old one. Once you copy over your API definitions, they should be identical. However, pay close attention to these kubebuilder [Markers][markers]:
```

Comment:
Should we not say that the files are very similar? In POV, since the markers changed we cannot say that they are identical.

Reply:
Could you please revert or improve this one?

Reply:
Sure, I can review this. While I rebase.

Reply:
I just pushed up a clarification. Let me know if that is better.